heareval/tasks/coughvid.py

```diff
@@ -389,7 +398,7 @@ def name(self):
     # the finalized top-level task directory
     @property
     def workdir(self):
-        return config.TASKNAME
+        return os.path.join("tasks", config.TASKNAME)
```

Comment:
I'm not sure what directory structure we want exactly, but at least all tasks should go into some directory.